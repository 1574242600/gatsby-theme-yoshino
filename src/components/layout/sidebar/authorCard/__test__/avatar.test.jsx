import React from "react";
import renderer from "react-test-renderer";
import Avatar from "../avatar";

describe("Avatar 组件", () => {

    it("快照", () => {
        const tree = renderer
            .create(<Avatar src={ "/avatar.jpg" } width={ 100 } height={ 100 }/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

});
