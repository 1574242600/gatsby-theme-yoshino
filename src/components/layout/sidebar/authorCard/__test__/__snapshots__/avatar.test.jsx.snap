// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar 组件 快照 1`] = `
<div
  className="avatar textCenter"
>
  <img
    className="avatarCircle lazyloading"
    height={100}
    src="/avatar.jpg"
    width={100}
  />
</div>
`;
